src/routes/User/Login.js

```diff
@@ -36,6 +36,10 @@ export default class LoginPage extends Component {
     this.ChildRef = React.createRef();
   }
 
+  componentDidMount() {
+    this.ChildRef.current.handleChange();
+  }
+
```

Comment:
What these changes for?

Reply:
The verification code is automatically refreshed once before the page loads

Reply:
Otherwise, it will always be the default

Reply:
Got it. You should separate different pr next time. : )